packages/block-library/src/heading/style.scss

```diff
@@ -1,4 +1,4 @@
-:root {
+.wp-gs {
```

Comment:
I was thinking we'd only enqueue the styles if theme declared support. As a temporary thing, scoping to `.wp-gs` is clearer and should be fine, though.

Reply:
yeah, this felt like a simpler way to do it. I don't feel strongly.

Reply:
As long as it doesn't lead people to start using this class to override things.

Reply:
I've slept on this and now I feel more inclined to avoid the class. Granted, it's going to be a bit more convoluted _for us to implement_, but I think it'll be a lot clearer _for theme authors_ not to have it. It'll avoid any confusion people may have with the class.

Although the plugin is a place for exploration there are some hosts that are already using it for production, so this looks like the kind of thing that can be difficult to get rid of if people start using it. I'd rather be conservative.